src/workerd/api/data-url.c++

```diff
@@ -11,18 +13,6 @@ kj::Maybe<DataUrl> DataUrl::tryParse(kj::StringPtr url) {
   return kj::none;
 }
 
-static constexpr kj::FixedArray<uint8_t, 256> ascii_whitespace_table = []() consteval {
-  kj::FixedArray<uint8_t, 256> result{};
-  for (uint8_t c: {0x09, 0x0a, 0x0c, 0x0d, 0x20}) {
-    result[c] = true;
-  }
-  return result;
-}();
-
-constexpr bool isAsciiWhitespace(uint8_t c) noexcept {
```

Comment:
Conceptually, this could affect performance – `isAsciiWhitespace()` does a single table lookup so having it defined in a different source file will make using it much slower based on needing a function call now. The compiler may or may not be able to inline it properly if LTO is enabled (such as with the downstream release build), so this is non-blocking.

Reply:
I moved the implementation to encoding.h and made it constexpr again. Since it is an extremely small fixed array, I don't think it would have any negative impact, but let me know if you disagree. I can change it back without any hassle.